lib/utils/aeon-urls.js

```diff
@@ -37,7 +26,8 @@ const aeonUrlForItem = async (esItem) => {
 
   const baseUrl = aeonBaseUrl + '/aeon/Aeon.dll?'
 
-  const sierraBib = esItem.item.bibs() && esItem.item.bibs().length ? esItem.item.bibs()[0] : null
+  const sierraItem = esItem.item
```

Comment:
while we're in the business of clearing up this ternary logic, maybe `sierraItem.bibs()?.length` will assist?

Reply:
Good call

Reply:
Fixed